installer/client/src/components/InstallerLayout.vue

```diff
@@ -36,8 +47,23 @@
 </template>
 
 <script>
+import InstallerSteps from "@/components/InstallerSteps.vue";
+
 export default {
   name: "InstallerLayout",
+  components: {
+    "installer-steps": InstallerSteps,
+  },
+  props: {
```

Comment:
@RajithaKumara @SandeepaOHRM need to add a mechanism to pass the `steps`, currently a flat string array is expected. `currentStep` is what should be highlighted. (counting from 0)

```
steps: [
    "Welcome",
    "Database Information",
    "System Check",
    "Add-on Check",
    "Version Details",
    "Upgrade",
    "Completion",
],
currentStep: 3, // Add-on Check
```